src/backend/lima.ts

```diff
@@ -1671,7 +1671,13 @@ CREDFWD_URL='http://${ hostIPAddr }:${ stateInfo.port }'
         const status = await this.status;
 
         if (defined(status) && status.status === 'Running') {
-          await this.execCommand({ root: true }, '/sbin/rc-service', '--ifstarted', 'k3s', 'stop');
+          if (this.cfg?.enabled) {
+            try {
+              await this.execCommand({ root: true }, '/sbin/rc-service', '--ifstarted', 'k3s', 'stop');
```

Comment:
This should probably have `expectFailure: true` in the options (which just silences things in the log, since we're logging it later anyway).